apps/src/lib/node/ledger/tendermint_node.rs

```diff
@@ -141,9 +149,9 @@ fn monitor_process(
     kill_switch: Sender<bool>,
```

Comment:
For consistency, `kill_switch` can be replaced with `abort_sender`

Reply:
done